src/appleseed/renderer/modeling/object/meshobject.cpp

```diff
@@ -37,6 +37,7 @@
 
```

Comment:
I prefer to keep alpha maps in MeshObject for now. If we find a good reason to support them in other object types, we can do it in a backwards compatible way without problems.